projects/admin/src/app/routes/documents-route.ts

```diff
@@ -125,8 +125,8 @@ export class DocumentsRoute extends BaseRoute implements RouteInterface {
               'language',
```

Comment:
I would put:

```typescript
            aggregationsOrder: [
              'document_type',
              'organisation',
              'language',
              'year',
              'author',
              'subject',
              'genreForm',
              'fiction',
              'intendedAudience',
              'acquisition',
              'status',
            ],
            aggregationsExpand: [
              'document_type',
              'organisation',
              'language',
              'year',
              'author',
              'subject',
            ],
```

That function in aggregationsExpand seems useless. @jma what do you think?

Reply:
@PascalRepond @jma This code has nothing to do with this PR. If it is necessary to correct, a new PR must be created to explain the reasons for the change.